chainer/utils/sparse.py

```diff
@@ -32,10 +43,20 @@ def __init__(self, data, row, col, shape, requires_grad=False):
             raise ValueError('length of shape must be 2.')
         if not (shape[0] > 0 and shape[1] > 0):
             raise ValueError('numbers in shape must be greater than 0.')
+        if not (order in ('C', 'F', 'auto', None)):
```

Comment:
A small detail but I don't think you need the outer parenthesis.

Reply:
Thanks! will remove the outer parenthesis.